ntbs-service/package.json

```diff
@@ -15,9 +15,9 @@
     "@babel/core": "7.8.4",
     "@babel/preset-env": "7.8.4",
     "accessible-autocomplete": "2.0.2",
-    "array-from-polyfill": "1.0.1",
     "axios": "0.19.2",
     "babel-loader": "8.0.6",
+    "css-vars-ponyfill": "2.2.1",
```

Comment:
Remove the carat please.
I think the package is also really stupidly named... it looks like a typo (not a criticism on the use of it though).

Reply:
there is no caret on the commit I can see, are you sure you're viewing the latest commit?